BragiCode/src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.controls.StrictFollower;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+
+public class ShooterSubsystem extends SubsystemBase {
+  public TalonFX mShooterMotorLeft;
+  public TalonFX mShooterMotorRight;
+
+  public ShooterState mShooterState;
+
+  private Timer mTimer = new Timer();
+
+  public enum ShooterState {
+  S_Shoot, S_WaitingForFeeder
+}
+
+  /** Creates a new ShooterSubsystem. */
+  public ShooterSubsystem() {
+    mShooterMotorLeft = new TalonFX(Constants.ShooterConstants.kShooterMotorLeft);
+    mShooterMotorRight = new TalonFX(Constants.ShooterConstants.kShooterMotorRight);
+
+    mShooterMotorRight.setControl(new StrictFollower(mShooterMotorLeft.getDeviceID()));
+
+
+  }
+
+  public void RunShooterState(){
+      switch (mShooterState){
+        case S_WaitingForFeeder:
+          WaitingForFeeder();
+          break;  
+        case S_Shoot:
+          Shoot();
+          break;
+      }
+
+    }
+
+    public void WaitingForFeeder(){
+      mShooterMotorLeft.set(0.5);
+    }
+
+    public void Shoot() {
+      mTimer.start();
+      if(mTimer.hasElapsed(5)){
+        mTimer.stop();
+        mTimer.reset();
+        mShooterState = ShooterState.S_WaitingForFeeder;
+      }
+    }
+
+    public double ShooterVelo(TalonFX motorFx){

```

Comment:
ShooterVelo looks like an internal function for this class. ShooterVelo should be private and static.

```suggestion
    private static double ShooterVelo(TalonFX motorFx){

```